app/models/input_file.rb

```diff
@@ -36,4 +36,8 @@ def s3_source_check
   def file_path
     File.join(sample.sample_path, 'fastqs', name)
   end
+
+  def type
```

Comment:
type seems to be a sensitive reserve word. Use file_type instead?

Reply:
ok